README.md

```diff
@@ -1 +1,194 @@
-# hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] I have noticed that you are missing the image in your directory, kindly consider adding your image and in addition, the image used should not be the one provided by Microverse. Consider having a customized logo instead ☺️.